mlir/include/mlir/Dialect/LLVMIR/NVVMOps.td

```diff
@@ -2101,6 +2101,23 @@ def NVVM_WgmmaMmaAsyncOp : NVVM_Op<"wgmma.mma_async",
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// NVVM breakpoint Op
+//===----------------------------------------------------------------------===//
+
+def NVVM_Breakpoint : NVVM_Op<"breakpoint"> {
```

Comment:
Why do we name OP as breakpoint while llvm intrinsic is debugtrap?

Reply:
Because it maps to the PTX `breakpoint` instruction (http://sw-mobile-docs/CUDA/GPGPU/parallel-thread-execution/index.html#miscellaneous-instructions-brkpt) and `debugtrap` looked ambigous given all other Ops maps 1-to-1 to PTX instructions